core/src/test/java/com/orange/ouds/core/component/OudsCheckboxItemTest.kt

```diff
@@ -30,7 +30,7 @@ internal class OudsCheckboxItemTest {
         }
 
         @Composable
-        override fun Snapshot(darkThemeEnabled: Boolean) {
+        override fun Snapshot(darkThemeEnabled: Boolean, highContrastModeEnabled: Boolean) {
             PreviewOudsCheckboxItem(
```

Comment:
As discussed together, maybe we should display high contrast mode preview for `OudsCheckboxItem` and add snapshot tests.

Moreover, the focus does not appear on hovered items in high contrast mode.